src/styles/app-button/AppButton.scss

```diff
@@ -0,0 +1,139 @@
+$spinner-size: 1.5rem;
+$spinning-animation: spinning 1.2s cubic-bezier(0.5, 0, 0.5, 1) infinite;
+
+.app-button {
+  display: flex;
+  justify-content: center;
+  transition: all 0.3s ease;
+
+  &--primary {
+    background: #000;
+    border: 1px solid #000;
+    color: #fff;
+
+    &:hover {
+      opacity: 0.8;
+    }
+
+    &:focus,
```

Comment:
Let's use `:focus-within` instead of `:focus` as we want trigger focus only via keyboard

Reply:
Quite interesting, I'll keep it in mind. Fixed

Reply:
But `:focus-within`  doesn't trigger focus only via keyboard, so I'd recommend to use `:focus-visible`